stress: failed test in cockroach/gossip/gossip.test: TestGossipNoForwardSelf

Binary: cockroach/static-tests.tar.gz sha: https://github.com/cockroachdb/cockroach/commits/7330af43411417dc60927b5a4cee0ee715b2cadc

Stress build found a failed test:

```
=== RUN   TestGossipNoForwardSelf
W160422 03:47:03.130795 gossip/gossip.go:892  not connected to cluster; use --join to specify a connected node
W160422 03:47:03.132224 gossip/gossip.go:892  not connected to cluster; use --join to specify a connected node
W160422 03:47:03.135124 gossip/gossip.go:892  not connected to cluster; use --join to specify a connected node
W160422 03:47:03.136453 gossip/gossip.go:892  not connected to cluster; use --join to specify a connected node
W160422 03:47:03.142374 gossip/gossip.go:892  not connected to cluster; use --join to specify a connected node
I160422 03:47:03.144283 gossip/server.go:185  refusing gossip from node 5 (max 3 conns); forwarding to 4 ({tcp 127.0.0.1:33951})
I160422 03:47:03.144449 gossip/client.go:86  closing client to node 1 (127.0.0.1:36677): gossip/client.go:170: received forward from node 1 to 4 (127.0.0.1:33951)
W160422 03:47:03.144681 gossip/gossip.go:892  not connected to cluster; use --join to specify a connected node
I160422 03:47:03.146569 gossip/server.go:185  refusing gossip from node 6 (max 3 conns); forwarding to 4 ({tcp 127.0.0.1:33951})
I160422 03:47:03.149940 gossip/client.go:86  closing client to node 1 (127.0.0.1:36677): gossip/client.go:170: received forward from node 1 to 4 (127.0.0.1:33951)
W160422 03:47:03.151553 gossip/gossip.go:892  not connected to cluster; use --join to specify a connected node
I160422 03:47:03.157182 gossip/server.go:185  refusing gossip from node 7 (max 3 conns); forwarding to 4 ({tcp 127.0.0.1:33951})
I160422 03:47:03.157475 gossip/client.go:86  closing client to node 1 (127.0.0.1:36677): gossip/client.go:170: received forward from node 1 to 4 (127.0.0.1:33951)
--- FAIL: TestGossipNoForwardSelf (0.04s)
	client_test.go:46: listen tcp 127.0.0.1:0: listen: address already in use
```

Run Details:

```
70 runs so far, 0 failures, over 5s
149 runs so far, 0 failures, over 10s
230 runs so far, 0 failures, over 15s
311 runs so far, 0 failures, over 20s
388 runs so far, 0 failures, over 25s
468 runs so far, 0 failures, over 30s
548 runs so far, 0 failures, over 35s
626 runs so far, 0 failures, over 40s
709 runs so far, 0 failures, over 45s
789 runs so far, 0 failures, over 50s
870 runs so far, 0 failures, over 55s
947 runs so far, 0 failures, over 1m0s
948 runs completed, 1 failures, over 1m0s
FAIL
```
Please assign, take a look and update the issue accordingly.

--- a/gossip/gossip_test.go
+++ b/gossip/gossip_test.go
@@ -134,7 +134,7 @@ func TestGossipNoForwardSelf(t *testing.T) {
 		})
 	}
 
-	const numClients = 50
+	numClients := len(peers) * 2
 	disconnectedCh := make(chan *client)
 	numFailedConns := 0
 

